### DISCUSSION TITLE
找回 UID 后如何恢复高级账户资格

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#1958] [高级账户]
> 此方式适用于：无法登录高级账户，但已找回 UID 的用户，[点击这里查看如何找回 UID](https://github.com/Kenshin/simpread/discussions/1571)

## 方案一

> 首选此方式，此方式仅限 简悦 2.2.0 版（含以上）。

选项页 → 账户 → 账户信息，按照下图修改即可。

![image](https://user-images.githubusercontent.com/81074/120882653-bcd5cc00-c60b-11eb-9628-427b0dec1aba.png)

## 方案二

> 简悦 2.2.0 以上版本使用。

在需要登录高级账户的浏览器 → 安装简悦 → 打开选项页 → 导出配置文件 → 用任意文本编辑器（打开） → 寻找并替换其中 uid 为正确的 → 导入此配置文件 → 右上角 icon → 仅授权当前设备

## 方案三

在需要登录高级账户的浏览器 → 安装简悦 → 打开选项页的账户选项卡 → 找回 UID → 自动找回

PS：此方式虽然简单，但会浪费一次找回机会（共计三次）

## 关联阅读

- [绑定高级账户](https://github.com/Kenshin/simpread/discussions/1573)

- [在新设备上登录](https://github.com/Kenshin/simpread/discussions/1574)

- [使用授权登录](https://github.com/Kenshin/simpread/discussions/1570)

- [取消授权](https://github.com/Kenshin/simpread/discussions/1575)

- [重新授权](https://github.com/Kenshin/simpread/discussions/1576)

- [找回丢失的 UID （高级账户资格）](https://github.com/Kenshin/simpread/discussions/1571)

- [找回 UID 后如何恢复高级账户资格](https://github.com/Kenshin/simpread/discussions/1958)

- [重新找到 License（兑换码）](https://github.com/Kenshin/simpread/discussions/2756)

- [安装和配置同步助手](https://github.com/Kenshin/simpread/discussions/2754)

**Comment by heartacker** [comment]
作为一个普通用户，都不知道咋弄。

**Reply by Kenshin** [reply]
方案一只需要一步操作。
https://github.com/Kenshin/simpread/discussions/1571 也是一次操作找回。

PS：接下来将会有更简单的方案。

**Comment by lanjuzi** [comment]
原来授权过的设备正常拥有高级权限，今天授权新设备的时候怎么都不成功，同步坚果云、手动备份恢复都失败了，折腾了很久。最后找到了以前备份的文件，对比后发现 uid 和已经授权的设备上显示的不一样，在新设备替换过来后正常授权成功。

**Reply by Kenshin** [reply]
哈哈，恭喜自救成功。

> 原来授权过的设备正常拥有高级权限

可以试试更简单的方式：用这个已经授权的设备的配置文件导入新设备就可以。

**Comment by CPCer** [comment]
建议写在wiki里，这样看懂了的用户可以参与完善帮助文档

**Reply by Kenshin** [reply]
谢谢建议，会考虑的。

**Comment by youying0356** [comment]
从本地导入配置文件后恢复高级账户资格，显示由于后台异常导致恢复购买失败，请稍后再试

**Reply by Kenshin** [reply]
应该是你的设置造成的，解决方案 https://github.com/Kenshin/simpread/discussions/1567

**Reply by youying0356** [reply]
确实是扩展设置的问题，已经解决了，谢谢！

**Reply by Kenshin** [reply]
okay 解决就好

PS：未防止你的信息泄流，我把你的截图删了 😄
